helm/sematic-server/templates/configmap.yaml

```diff
@@ -25,6 +25,6 @@ data:
 {{ if .Values.server.sematic_dashboard_url }}
   SEMATIC_DASHBOARD_URL: {{ .Values.server.sematic_dashboard_url | quote }}
 {{ end }}
-{{ if .Values.sematic.socket_io_address }}
-  SEMATIC_SOCKET_IO_ADDRESS: {{ .Values.sematic.socket_io_address }}
+{{ if .Values.deployment.socket_io.dedicated }}
+  SEMATIC_SOCKET_IO_ADDRESS: {{ .Values.deployment.socket_io.address }}
```

Comment:
Can we construct this (at least a default in the situation where we created the service for it)?

Reply:
👍